src/julia.expmap

```diff
@@ -28,6 +28,9 @@
     save_arg_area_loc;
     u8_*;
     uv_*;
+    unw_*;
+    _Ux86_64_*;
+    _ULx86_64_*;
```

Comment:
`_Ux86_` and `_ULx86_` for 32bit, `_Uaarch64_` and `_ULaarch64_` for AArch64. Possibly also `_U_dyn_`.